### PR TITLE
new icons

### PR DESCRIPTION
**Pull request by cabutler10**

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...:"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"22","messages":"23","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"24","messages":"25","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"26","messages":"27","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"28","messages":"29","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/index.js",[],"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/elements/button/ButtonPage.jsx",[],"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/elements/notification/NotificationPage.jsx",[],"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/elements/tabs/TabsPage.jsx",[],"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/elements/spinner/SpinnerPage.jsx",[],"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/elements/tag/TagPage.jsx",[],"/Users/alyssabutler/Documents/ZLab/zeppelin-element-library/src/core/Element.js",[]]
```
Comment (byjs-dev) [inline .eslintcache:1]: Please put that into .gitignore and remove it from the PR. ;)

**Review by byjs-dev — CHANGES_REQUESTED** [review]
Could you please use "-" instead of "_" in the icon-/file name? Unless our CoP proposed solution changed. Sorry and thank you.

**Comment by cabutler10** [comment]
@byjs-dev i didn't notice, dashes have been updated!

**Review by byjs-dev — APPROVED** [review]
Thank you. Please just remove the .eslintchache and add it to the .gitignore file before merging. 
As next step I'd like to have the creation of the svg spritesheet automated and also an optimization for the svgs. We could use something like https://github.com/edenspiekermann/sprite.sh and https://github.com/svg/svgo for that. What do you think?

**Comment by cabutler10** [comment]
@byjs-dev i never use svg sprites so am super happy to have your expertise here. Would you be willing to take one the work to change the sprites?

**Comment by byjs-dev** [comment]
> @byjs-dev i never use svg sprites so am super happy to have your expertise here. Would you be willing to take one the work to change the sprites?

Sure, I'll add an issue with this topic.

**Pull request merged**